### ISSUE TITLE
Icon characters may be read by AT

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#1136]
Using a pseudo element on an active container may result in some newer combinations of browsers and AT exposing the font character used for the icon. All icons should be converted to be rendered inside aria-hidden containers to prevent this.
